Kudu.Core/Settings/EnvironmentSettingNames.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
The environment variables should be placed in `Common\Constants.cs`. If any of these is app settings, then it should be placed in `Kudu.Contracts\Settings\SettingsKeys.cs`.